src/Simulation/Algorithms/Kepler/NU2ANOM.hpp

```
@@ -0,0 +1,70 @@
+		double m = NAN;
+		
+
+		// Circular orbit (e = 0)
+		if (glm::abs(e) < R_EPSILON) {
+			e0 = nu;
+			m = nu;
+		}
+
+		// Elliptical orbit (0 < e < 1)
+		else if (e < 1.0 - R_EPSILON) {
+			double sin_e = (glm::sin(nu) * glm::sqrt(1.0 - e * e)) / (1.0 + e * glm::cos(nu));
```

Clang (14.0.6)

[warning] variable 'sin_e' is not initialized

(cppcoreguidelines-init-variables)